doc/fwutil/fwutil.md

```diff
@@ -158,20 +173,21 @@ In order to improve scalability and performance a modern network switches provid
 1. Non modular chassis platforms
 2. Modular chassis platforms
 
-Non modular chassis platforms may contain only one chassis.  
+Non modular chassis platforms may contain only one chassis.
 A chassis may contain it's own set of components.
 
-Modular chassis platforms may contain only one chassis.  
-A chassis may contain one or more modules and it's own set of components.  
+Modular chassis platforms may contain only one chassis.
+A chassis may contain one or more modules and it's own set of components.
 Each module may contain it's own set of components.
 
 Basically each chassis/module may contain one or more components (e.g., BIOS/CPLD/FPGA).
 
-SONiC platform API provides an interface for FW maintenance operations for both modular and  
-non modular chassis platforms. Both modular and non modular chassis platforms share the same platform API,  
+SONiC platform API provides an interface for FW maintenance operations for both modular and
+non modular chassis platforms. Both modular and non modular chassis platforms share the same platform API,
 but may have different implementation.
 
 SONiC FW utility uses platform API to interact with the various platform components.
+SONiC FW utility extends to support for the automatic firmware update based on <platform>_components.json and next reboot option.
```

Comment:
@sujinmkang IMHO, it's better to mention here that besides Platform API FW tools can also use plugin concept when all the auxiliary stuff is provided within a package (flash tools, scripts, etc.)